modules/ROOT/pages/salesforce/salesforce-composite-connector-reference.adoc

```diff
@@ -1,889 +1,1187 @@
-| Consumer Key a| String |  Consumer key for Salesforce connected app |  |x
-| Consumer Secret a| String |  Your application's client secret (consumer secret in Remote Access Detail). |  |x
-| Username a| String |  Username used to initialize the session |  |x
-| Password a| String |  Password used to authenticate the user |  |x
-| Security Token a| String |  User's security token. It can be omitted if your IP has been whitelisted on Salesforce |  |
-| Token Endpoint a| String |  URL pointing to the server responsible for providing the authentication token. According to Salesforce it should be `+https://login.salesforce.com/services/oauth2/token+`, or, if implementing for a community, `+https://acme.force.com/customers/services/oauth2/token+` (where acme.force.com/customers is your community URL). |  `+https://login.salesforce.com/services/oauth2/token+` |
-| Reconnection a| Reconnection |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
-| Host a| String |  Hostname of the proxy. If this property is not set, no proxy is used, otherwise a proxy ise used, but a proxy host must be specified. |  |
-| Port a| Number |  Port of the proxy. If host is set then this property must be set and cannot be a negative number. |  |
-| Username a| String |  Username used to authenticate against the proxy. If this property is not set then no authentication is going to be used against the proxy, otherwise this value must be specified. |  |
-| Password a| String |  Password used to authenticate against the proxy. |  |
+| Consumer Key a| String |  Consumer key for the Salesforce connected app. |  | x
+| Consumer Secret a| String |  Your app's client secret (consumer secret in remote access detail). |  | x
+| Username a| String |  Username used to initialize the session. |  | x
+| Password a| String |  Password used to authenticate the user. |  | x
+| Security Token a| String |  User's security token, which can be omitted if your IP has been white-listed on Salesforce. |  | 
+| Token Endpoint a| String |  URL pointing to the server responsible for providing the authentication token. According to Salesforce it should be `+https://login.salesforce.com/services/oauth2/token+`, or, if implementing for a community, `+https://acme.force.com/customers/services/oauth2/token+` (where `acme.force.com/customers` is your community URL). |  `+https://login.salesforce.com/services/oauth2/token+` | 
+| Reconnection a| <<Reconnection>> |  When an app is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+| Host a| String |  Hostname of the proxy. If this property is not set, then no proxy is used, otherwise a proxy is used, but a proxy host must be specified. |  | 
```

Comment:
Hostname of the proxy. If set, then a proxy host must be specified. If not set, no proxy is used.